distsys/20200218.md

```diff
@@ -0,0 +1,45 @@
+# 2020 February Meetup
+
+First meetup, yay!
+
+## Atendees:
+
+- [ignasi35](https://twitter.com/ignasi35): organizer
+- [@joaoalves](https://twitter.com/joaoqalves)
+- [@bakwrau](https://twitter.com/bakwrau)
+- [@gonzaloserrano](https://twitter.com/gonzaloserrano)
```

Comment:
We are all 4 organizers.

And, in my opinion, anyone who attends will be an organizer too.